### PR TITLE
bump to 1.0.1  note: this is also on pypi

### PR DESCRIPTION
**Pull request by gdestuynder**
https://pypi.python.org/pypi/authzerolib => 1.0.1

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def test_suite():
 setup(
     name="authzerolib",
     py_modules=['authzerolib'],
-    version="1.0.0",
+    version="1.0.1",
     author="Guillaume Destuynder",
     author_email="gdestuynder@mozilla.com",
     description=("A super simple and basic client lib for Auth0"),
```
Comment (andrewkrug) [inline setup.py]: please add install dependencies to setup.py

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
```
Comment (andrewkrug) [inline CHANGELOG.md]: <3 changelog standard

File: setup.py (hunk 1 above)
Comment (gdestuynder) [reply setup.py]: what if its dependencies-less ? ;-)

Comment (andrewkrug) [reply setup.py]: there's a way to declare test depends in the setup.py as well.  Then you can use `python setup.py test`

**Comment by andrewkrug** [comment]
No reason I see to block on the test depends stuff.  Let's find the *right* way to do that though.

**Pull request merged**